content/docs/community/faq.md

```diff
@@ -253,3 +253,21 @@ definitions, as that will lead to duplicates.
 # Does Cucumber-JVM support Kotlin?
 You can use [Cucumber-JVM](https://github.com/cucumber/cucumber-jvm) to write step definitions in Kotlin. Please have a look at the [Kotlin examples for cucumber-jvm](https://github.com/cucumber/cucumber-jvm/tree/master/kotlin-java8). 
 At the moment it is not possible to generate step definitions in Kotlin. The reason for this is that there is no Kotlin Backend implemented. If this is something you'd like to work on, there is [a request for one](https://github.com/cucumber/cucumber-jvm/issues/1520). There is also a request for a [native Kotlin implementation of Cucumber](https://github.com/cucumber/cucumber/issues/331).
+
+# Cucumber Can't Find my Step Definitions in IntelliJ IDEA
+In this instance, you need to configure a new run configuration in IntelliJ IDEA. 
+1. Click **Run** > **Edit Configurations** from the menu in IntellJ IDEA.
+2. Click the **+** icon on the top-left and type in _cucumber_. Select **Cucumber Java**.
+3. Create the configuration according to the [Run/Debug Configuration Cucumber Java](https://www.jetbrains.com/help/idea/run-debug-configuration-cucumber-java.html) instructions from JetBrains. 
+   
+If IntelliJ IDEA doesn't recognize the package with step definitions, you can specify it manually by entering the package name in the Glue field, for example _stepdefs_.
+
+For more information, please see [Run Cucumber Tests](https://www.jetbrains.com/help/idea/running-cucumber-tests.html) from JetBrains documentation.
+
+# Running Cucumber results in a stack trace containing 'Failed to instantiate public cucumber.runtime.java.JavaBackend or NoSuchMethodException'
```

Comment:
This one is a bit ambiguous from the POV of what is my question e.t.c.

Is this `Cucumber JVM Throws Error with a stacktrace containing .....`

Reply:
Hi @luke-hill, I don't know. I've used what information there was on the ticket (https://github.com/cucumber/docs.cucumber.io/issues/284) but I'm not sure what else I can add. If anyone has more information I can absolutely add it though.

Reply:
Sorry I mean can you frame this as a Frequently Asked question. It's a bit ambiguous at the moment if this is a problem or solution. I had a brief look at the issue you linked, so I think you should re-phrase this as.

> How do I fix Cucumber-JVM erroring with a stacktrace 'Failed to instantiate public cucumber.runtime.java.JavaBackend or NoSuchMethodException'?

Reply:
Ahhh, okay, was being slow :D will do!